terraform/providers.tf

```diff
@@ -9,12 +9,12 @@ provider "aws" {
   region = var.aws_region
 }
 
-# The provider used to create the role that can be assumed to do
-# everything the CI user needs to do in the staging account.
+# The provider used to create roles that can read certificates from a
+# production S3 bucket
```

Comment:
I think this comment should be updated to reflect what we _are_ reading.

Reply:
Good catch - fixed in https://github.com/cisagov/ansible-role-assessor-portal/pull/3/commits/4eb45c886c7b594b216996ff2e00db053b9f03ae.